src/features/MovieDetailsPage/Header/styled.js

```diff
@@ -0,0 +1,157 @@
+import styled from "styled-components";
+import { ReactComponent as StarIcon } from "./starIcon.svg";
+
+export const Block = styled.header`
+    display: flex;
+    justify-content: center;
+    max-width: 100%;
+    max-height: 770px;
+    background-color: ${({ theme: { colors } }) => colors.woodSmoke};
+`;
+
+export const Wrapper = styled.article`
+    position: relative;
+    display: flex;
+    max-width: 1370px;
+    flex-basis: 100%;
+    max-height: 770px;
+`;
+
+export const Poster = styled.img`
+    width: 100%;
+    padding: 0px 16px;
+    filter: brightness(80%);
+`;
+
+export const Shadow = styled.div`
+    position: absolute;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    height: 100%;
+    top: 0;
+    left: 0;
+    box-shadow: inset 0 0 60px 100px ${({ theme: { colors } }) => colors.woodSmoke};
+
+    @media (max-width: ${({ theme: { breakpoints } }) => breakpoints.tabletMax}px) {
+        box-shadow: inset 0 0 35px 60px ${({ theme: { colors } }) => colors.woodSmoke};
+    }; 
+
+    @media (max-width: ${({ theme: { breakpoints } }) => breakpoints.mobileMax}px) {
+        box-shadow: inset 0 0 20px 30px ${({ theme: { colors } }) => colors.woodSmoke};
+        width: 96%;
+        height: 100%;
+        left:2%;
+    }; 
+
+    @media (max-width: ${({ theme: { breakpoints } }) => breakpoints.mobileMin}px) {
+        box-shadow: inset 0 0 10px 20px ${({ theme: { colors } }) => colors.woodSmoke};
+        width: 95%;
+        height: 100%;
+        left:2.5%;
+    }; 
+`;
+
+export const Content = styled.div`
+    color: ${({ theme: { colors } }) => colors.white};
+    position: absolute;
+    padding: 0 24px 56px;
+    bottom: 0;
+    left: 0;
```

Comment:
zrobiłeś flexa, dałeś mu `justify-content: center`,  a teraz wszędzie dajesz `position: absolute`, trochę to bez sensu chyba 😅